bootstrap.py

```diff
@@ -1434,9 +1436,15 @@ def exec_service(service, command, failonerror=True):
 
         # Configure new proxy_id for Puppet (if not skipped), and OpenSCAP (if available and not skipped)
         if 'foreman' not in options.skip and 'puppet' not in options.skip:
-            print_running("Calling Foreman API to update Puppet master and Puppet CA for %s to %s" % (FQDN, options.foreman_fqdn))
-            update_host_capsule_mapping("puppet_proxy_id", smart_proxy_id, current_host_id)
-            update_host_capsule_mapping("puppet_ca_proxy_id", smart_proxy_id, current_host_id)
+            if puppet_master_smart_proxy_id is not None and puppet_ca_smart_proxy_id is not None:
```

Comment:
you can write this without the `is not None` part, which I find better readable
```suggestion
            if puppet_master_smart_proxy_id and puppet_ca_smart_proxy_id:
```